package.json

```diff
@@ -31,6 +32,7 @@
     "@types/sinon": "^10.0.14",
     "expect": "^29.5.0",
     "mocha": "^10.2.0",
+    "nyc": "^17.0.0",
```

Comment:
Should we add nyc? Maybe we can add it in another PR and sync it with Coveralls, what do you think?